test/cxx/Core/ApplicationPool/ProcessTest.cpp

```diff
@@ -238,4 +243,15 @@ namespace tut {
 				&& contents.find("stdout and err 4\n") != string::npos;
 		);
 	}
+
+    TEST_METHOD(6) {
+        set_test_name("Test generation is inherited from pool at construction time");
+	    // this test is pointless b/c the process is made at line 123 of this file, not in the regular codebase
```

Comment:
This test is indeed pointless when done from ProcessTest.cpp. The behavior is not something that exists solely in Process, but is something that exists as part of the interaction between Group and Process. Therefore, this test should be removed.

Test 15 in PoolTest is good enough.